### DISCUSSION TITLE
添加子域名

### DISCUSSION THREAD

**Discussion by ghwnode02** [dreamhunter2333/cloudflare_temp_email#395] [General]
## 部署方式

- [ ] cli 部署
- [√ ]  用户界面部署

## 浏览器环境
edge

**Comment by ghwnode02** [comment]
在域名的电子邮件路由添加一个子域subdomain.example.com, 这可以让cf自动生成一些DNS记录

添加/修改DNS记录

spf记录复制粘贴文档的就好，即v=spf1 include:_spf.mx.cloudflare.net include:relay.mailchannels.net ~all
_mailchannels.subdomain的内容复制原本_mailchannels的内容，subdomain可以写全但cf也会帮你自动缩写
mailchannels._domainkey.subdomain的内容复制原本mailchannels._domainkey的内容
_dmarc.subdomain的内容复制原本_dmarc的内容
一句话概括下，就是你的example.com的DNS有哪些TXT的解析记录，给要用来发邮件的子域名复制一份，名称里面就带上子域名。
把子域名添加到workers变量的DOMAINS里，部署即可.

按照上述的配置方法，MX和TXT，spf记录cloud flare会自动生成，但是DKIM我是用的resend，需要使用子域名添加到resend获取token吗？还是使用之前的DKIM就可以。

把子域名添加到workers变量的DOMAINS里
这个地方workers变量添加子域名的格式是怎样的？
我测试了["xxx.example.com"]["bbb.example.com"]和["xxx.example.com","bbb.example.com"]都无法使用

**Comment by dreamhunter2333** [comment]
应该 resend 添加域名的时候也要用子域名

**Comment by ghwnode02** [comment]
我cloudns的域名测试了一下子域名好像无法添加到resend

**Comment by dreamhunter2333** [comment]
> 我cloudns的域名测试了一下子域名好像无法添加到resend

那看起来 resend 无法使用子域名了，我文档中说明下这点

**Comment by ghwnode02** [comment]
添加另外一个域名，要怎么配置

**Comment by dreamhunter2333** [comment]
> 添加另外一个域名，要怎么配置

domains 是个列表，增加元素就可以

**Comment by dreamhunter2333** [comment]
建议使用主域名发送邮件，子域名可以用来接收邮件

**Comment by unobject** [comment]
可以的 我发现是 send@abc.yourfdomain.xyz 这种格式

**Comment by Albertjasson22** [comment]
I need a way to contact the developer or website owner as soon as possible.
